src/cmd/node/rpc/get_module_abi.rs

```diff
@@ -0,0 +1,38 @@
+use anyhow::{Context, Result};
+use clap::Parser;
+use jsonrpsee::core::client::ClientT;
+use jsonrpsee::http_client::HttpClientBuilder;
+use jsonrpsee::rpc_params;
+
+/// Estimate gas for publishing modules.
+#[derive(Parser, Debug)]
+#[clap(about = "Get a move module's ABI")]
+pub struct GetModuleAbi {
+    #[clap(short, long, help = "Address of the module")]
+    address: String,
```

Comment:
Unrelated to your change:

I think we should change this to `account_id` since the SS58 account will be the most common input here. You can leave a TODO here, otherwise, we would need to update the substrate side - but we can do that later
```suggestion
    account_id: String,
```

Reply:
Yes, I also realised that, but decided to postpone it, because I was not absolutely sure, if it is always the `accound_id` and because I wanted to have all the paramters name's unique down the chain...

Reply:
> but decided to postpone it,

Nice! Let's do that in some future PR.